database/postgres/definition.yaml

```
@@ -23,7 +23,7 @@ metadata:
         - name: networkRef
           controlType: singleInput
           type: string
-          path: ".spec.parameters.networkRef.name"
+          path: ".spec.parameters.networkRef.id"
           title: Network Ref
           description: Network fabric to connect the database to
           default: network
```

Should this match the network.id from examples/network.yaml?

```suggestion
          default: platform-ref-aws-network
```